pkg/counter/cache.go

```diff
@@ -263,13 +293,17 @@ func initializeFromResources(cl client.Client, namespace string) error {
 	}
 	reset()
 	for _, usersignup := range usersignups.Items {
-		if value, exists := usersignup.Annotations[toolchainv1alpha1.UserSignupActivationCounterAnnotationKey]; exists {
-			_, err := strconv.Atoi(value) // let's make sure the value is actually an integer
+		activations, activationsExists := usersignup.Annotations[toolchainv1alpha1.UserSignupActivationCounterAnnotationKey]
+		if activationsExists { // (DEPRECATED)
```

Comment:
why is this line deprecated?

Reply:
that's probably a left-over comment, I removed it in b508230